templates/en/codelist.rst

```diff
@@ -68,9 +68,12 @@ Codes
      - Public Database?{% endif %}
 
    {% for codelist_item in codelist_json.data %}
-       {% if codelist_item.status == 'withdrawn' %} .. rst-class:: withdrawn
+       {% if codelist_item.status == 'withdrawn' %} 
+       .. rst-class:: withdrawn
+   * - {{codelist_item.code + " (withdrawn)"}}
```

Comment:
So this all looks good to me.

One really really small thing that I think it’s totally fine to ignore… Instead of:

```
* - {{codelist_item.code + " (withdrawn)"}}
```

…you could do:

```
* - {{codelist_item.code}} (withdrawn)
```

…just to avoid some unnecessary string concatenation. But honestly, I think it’s fine and probably not worth changing.